ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/CloseButton.kt

```diff
@@ -25,7 +27,11 @@ fun BoxScope.CloseButton(
                 onClick = onClick,
                 enabled = actionInProgress.not(),
             ) {
-                Icon(painter = painterResource(id = R.drawable.close), contentDescription = null)
+                Icon(
+                    painter = painterResource(id = R.drawable.close),
+                    contentDescription = null,
+                    tint = color ?: Color.Black,
```

Comment:
Can we default this to `LocalContentColor.current`? This is what was before, so we don't change the color for users inadvertently.